README.md

```diff
@@ -53,7 +53,7 @@ any new files in the repository.
 Make your own customizations
 ----------------------------
 
-Put your customizations in dotfiles appended with `.local`:
+Put your customizations in `~/dotfiles-local` appended with `.local`:
```

Comment:
I think this is too prescriptive in regards to path and directory. How about just stating that customizations can be up in **a directory** and added to `rcrc`?